parsers/US_NY.py

```diff
@@ -8,13 +8,20 @@
 import pandas as pd
 
 mapping = {
-           'Dual Fuel': 'unknown',
+           'Dual Fuel': 'gas',              # NYISO states "Power produced by facilities capable of using natural gas and/or other fuels"
```

Comment:
Naive question: does Dual Fuel only mean gas + oil? Could it be gas + biomass? or coal?

Reply:
for NY's purposes it is only gas or oil.   They can't feed coal or biomass into the units.

In the SE US we get into "dual fuel" coal & biomass.  Usual location is a paper mill, with onsite generation.  They feed coal, tree bark and "black liquor" (sap & tannins), any recovered or unusable byproducts into the boilers.

Reply:
not self-promotion, but here is [me] from 380ft above a paper factory (from 1 of 3 exhaust stacks) in South Carolina - where they do the above (coal, biomass & black liquor)  https://www.instagram.com/p/bG4AoEtl-b/?taken-by=hansworldtravels